Index: nil/nil/nil/anim_helpers.h
===================================================================
--- nil/nil/nil/anim_helpers.h
+++ nil/nil/nil/anim_helpers.h	2009-05-16 17:30:51.000000000 +0200
@@ -16,6 +16,7 @@
 #include "vector"
 #include "map"
 #include "string"
+#include <cstring>
 #include "graphic.h"
 
 //-------------------------------------------------------------------
@@ -238,4 +239,4 @@
 };
 
 
-#endif
\ No newline at end of file
+#endif
